Not sure where these failures in window come from

[mdanalysis-CI (Azure_Tests Win-Python38-32bit-full)](https://github.com/MDAnalysis/mdanalysis/pull/3951/checks?check_run_id=9956143561) fails [TestH5MDWriterBaseAPI.test_write_trajectory_universe ](https://dev.azure.com/mdanalysis/mdanalysis/_build/results?buildId=4820&view=logs&j=86f9dbba-a936-58d9-7027-9855b63ee954&t=eae8c275-7e36-56e4-0ff0-05635fb44905&l=281)
```
____________ TestH5MDWriterBaseAPI.test_write_trajectory_universe _____________
[gw0] win32 -- Python 3.8.10 C:\hostedtoolcache\windows\Python\3.8.10\x86\python.exe

self = <MDAnalysisTests.coordinates.test_h5md.TestH5MDWriterBaseAPI object at 0x10D5B148>
                            velocities=True, forces=True) as w:
>               for ts in universe.trajectory:

D:\a\1\s\testsuite\MDAnalysisTests\coordinates\test_h5md.py:135: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
C:\hostedtoolcache\windows\Python\3.8.10\x86\lib\site-packages\mdanalysis-2.4.0.dev0-py3.8-win32.egg\MDAnalysis\coordinates\base.py:795: in __iter__
    self._reopen()
C:\hostedtoolcache\windows\Python\3.8.10\x86\lib\site-packages\mdanalysis-2.4.0.dev0-py3.8-win32.egg\MDAnalysis\coordinates\H5MD.py:747: in _reopen
    self.close()
C:\hostedtoolcache\windows\Python\3.8.10\x86\lib\site-packages\mdanalysis-2.4.0.dev0-py3.8-win32.egg\MDAnalysis\coordinates\H5MD.py:728: in close
    self._file.close()
C:\hostedtoolcache\windows\Python\3.8.10\x86\lib\site-packages\h5py\_hl\files.py:435: in close
    file_list = [x for x in file_list if h5i.get_file_id(x).id == self.id.id]
C:\hostedtoolcache\windows\Python\3.8.10\x86\lib\site-packages\h5py\_hl\files.py:435: in <listcomp>
    file_list = [x for x in file_list if h5i.get_file_id(x).id == self.id.id]
h5py\_objects.pyx:54: in h5py._objects.with_phil.wrapper
    ???
h5py\_objects.pyx:55: in h5py._objects.with_phil.wrapper
    ???
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

>   ???
E   ValueError: Not an id of a file object (not an ID of a file object)

h5py\h5i.pyx:114: ValueError
```

I am letting it *re-run*, maybe that's a transient issue.

--- a/package/CHANGELOG
+++ b/package/CHANGELOG
@@ -89,10 +89,11 @@ Changes
   * adding element attribute to TXYZParser if all atom names are valid element symbols (PR #3826)
 
 Deprecations
+  * Deprecated analysis.align.sequence_alignment() for removal in 3.0 (#3950)
   * Add deprecation warning for `timestep` copying in DCDReader
     (Issue #3889, PR #3888)
   * Add deprecation warning for inclusive stop indexing in
-    MemoryReader.timeseries (#PR 3894)
+    MemoryReader.timeseries (PR #3894)
 
 
 08/29/22 IAlibay, PicoCentauri, orbeckst, hmacdope, rmeli, miss77jun, rzhao271,

--- a/package/MDAnalysis/analysis/align.py
+++ b/package/MDAnalysis/analysis/align.py
@@ -202,6 +202,7 @@ from MDAnalysis.exceptions import SelectionError, SelectionWarning
 import MDAnalysis.analysis.rms as rms
 from MDAnalysis.coordinates.memory import MemoryReader
 from MDAnalysis.lib.util import get_weights
+from MDAnalysis.lib.util import deprecate   # remove 3.0
 
 from .base import AnalysisBase
 
@@ -967,7 +968,9 @@ class AverageStructure(AnalysisBase):
         warnings.warn(wmsg, DeprecationWarning)
         return self.results.rmsd
 
-
+@deprecate(release="2.4.0", remove="3.0",
+           message="See the documentation under Notes how directly use"
+                   "Bio.Align.PairwiseAligner with ResidueGroups.")
 def sequence_alignment(mobile, reference, match_score=2, mismatch_penalty=-1,
                        gap_penalty=-2, gapextension_penalty=-0.1):
     """Generate a global sequence alignment between two residue groups.

--- a/package/MDAnalysis/analysis/align.py
+++ b/package/MDAnalysis/analysis/align.py
@@ -968,8 +968,9 @@ class AverageStructure(AnalysisBase):
         warnings.warn(wmsg, DeprecationWarning)
         return self.results.rmsd
 
+
 @deprecate(release="2.4.0", remove="3.0",
-           message="See the documentation under Notes how directly use"
+           message="See the documentation under Notes on how to directly use"
                    "Bio.Align.PairwiseAligner with ResidueGroups.")
 def sequence_alignment(mobile, reference, match_score=2, mismatch_penalty=-1,
                        gap_penalty=-2, gapextension_penalty=-0.1):
@@ -1066,6 +1067,7 @@ def sequence_alignment(mobile, reference, match_score=2, mismatch_penalty=-1,
     return AlignmentTuple(topalignment[0], topalignment[1],
                           topalignment.score,
                           0, max(reference.n_residues, mobile.n_residues))
+
 
 
 def fasta2select(fastafilename, is_aligned=False,

--- a/testsuite/MDAnalysisTests/analysis/test_align.py
+++ b/testsuite/MDAnalysisTests/analysis/test_align.py
@@ -542,6 +542,7 @@ class TestAlignmentProcessing(object):
         assert len(sel['reference']) == 30621, self.error_msg
         assert len(sel['mobile']) == 30621, self.error_msg
 
+
 class TestSequenceAlignmentFunction:
     # remove 3.0
 
@@ -565,7 +566,7 @@ class TestSequenceAlignmentFunction:
                      err_msg="reference sequence mismatch")
         assert mobile.residues.sequence(
             format="string") in seqB, "mobile sequence mismatch"
-        assert_almost_equal(score, 54.6)
+        assert score  == pytest.approx(54.6)
         assert_array_equal([begin, end], [0, reference.n_residues])
 
     def test_sequence_alignment_deprecation(self, atomgroups):
@@ -574,6 +575,7 @@ class TestSequenceAlignmentFunction:
                 "`sequence_alignment` will be removed in release 3.0.")
         with pytest.warns(DeprecationWarning, match=wmsg):
             align.sequence_alignment(mobile, reference)
+
 
 def test_alignto_reorder_atomgroups():
     # Issue 2977

--- a/testsuite/MDAnalysisTests/analysis/test_align.py
+++ b/testsuite/MDAnalysisTests/analysis/test_align.py
@@ -542,22 +542,38 @@ class TestAlignmentProcessing(object):
         assert len(sel['reference']) == 30621, self.error_msg
         assert len(sel['mobile']) == 30621, self.error_msg
 
-def test_sequence_alignment():
-    u = mda.Universe(PSF)
-    reference = u.atoms
-    mobile = u.select_atoms("resid 122-159")
-    aln = align.sequence_alignment(mobile, reference)
+class TestSequenceAlignmentFunction:
+    # remove 3.0
 
-    assert len(aln) == 5, "return value has wrong tuple size"
+    @staticmethod
+    @pytest.fixture
+    def atomgroups():
+        universe = mda.Universe(PSF)
+        reference = universe.atoms
+        mobile = universe.select_atoms("resid 122-159")
+        return reference, mobile
 
-    seqA, seqB, score, begin, end = aln
-    assert_equal(seqA, reference.residues.sequence(format="string"),
-                 err_msg="reference sequence mismatch")
-    assert mobile.residues.sequence(
-        format="string") in seqB, "mobile sequence mismatch"
-    assert_almost_equal(score, 54.6)
-    assert_array_equal([begin, end], [0, reference.n_residues])
+    @pytest.mark.filterwarnings("ignore:`sequence_alignment` is deprecated!")
+    def test_sequence_alignment(self, atomgroups):
+        reference, mobile = atomgroups
+        aln = align.sequence_alignment(mobile, reference)
 
+        assert len(aln) == 5, "return value has wrong tuple size"
+
+        seqA, seqB, score, begin, end = aln
+        assert_equal(seqA, reference.residues.sequence(format="string"),
+                     err_msg="reference sequence mismatch")
+        assert mobile.residues.sequence(
+            format="string") in seqB, "mobile sequence mismatch"
+        assert_almost_equal(score, 54.6)
+        assert_array_equal([begin, end], [0, reference.n_residues])
+
+    def test_sequence_alignment_deprecation(self, atomgroups):
+        reference, mobile = atomgroups
+        wmsg = ("`sequence_alignment` is deprecated!\n"
+                "`sequence_alignment` will be removed in release 3.0.")
+        with pytest.warns(DeprecationWarning, match=wmsg):
+            align.sequence_alignment(mobile, reference)
 
 def test_alignto_reorder_atomgroups():
     # Issue 2977

